README.md

```diff
@@ -1,19 +1,19 @@
-# LiveChat Platform Docs
+# Platform Docs
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/64c9d8bf-8fe6-4481-91d9-4258c3b9ae91/deploy-status)](https://app.netlify.com/sites/livechat-public-docs/deploys)
 
 Hello 👋!
 
-This is the for all of the [LiveChat Platform documentation](https://developers.livechat.com/docs/).
+This is the repository for the [Platform documentation](https://developers.livechat.com/docs/).
 Our documents are written in the [MDX format](https://mdxjs.com/), so pay attention to our custom tags when contributing to the docs.
 
 ## Who we are
 
 Behind LiveChat, Inc. there is a team of [passionate people](https://www.livechat.com/team/) building online customer service software with online chat, help desk software, chatbot, and web analytics capabilities.
```

Comment:
Are we leaving `LiveChat` or changing to `Text` like in Docs?

Reply:
Okay, I wasn't sure when we implement the changes to the naming, so I preferred to ask😉